generators/app/templates/root/_package.json

```diff
@@ -40,7 +40,7 @@
     "body-parser": "^1.19.0",
     "boom": "^7.3.0",
```

Comment:
Note: Boom has been depretated. We need to migrate to `@hapi/boom`

Reply:
thanks and done!